opdracht1.js

```diff
@@ -0,0 +1,43 @@
+// Opdracht 1 - Array Methoden
+// Opdracht 1a: Gebruik een array-methode om een array te maken met alle tv-type namen. Log de uitkomst in de console.
+
+const tvType = inventory.filter((tv) => {
+    return tv.type;
+})
+
+console.log(tvType);
+
+
+//     Opdracht 1b: Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht zijn. Log de uitkomst in de console.
+
+const tvSoldOut = inventory.filter((tv) => {
+    if (tv.originalStock === tv.sold)
+        return inventory;
+})
```

Comment:
.filter verwacht een functie die een boolean teruggeeft
```suggestion
const tvSoldOut = inventory.filter(tv => tv.originalStock === tv.sold);
```